controllers/harborcluster_controller.go

```diff
@@ -45,12 +47,15 @@ type HarborClusterReconciler struct {
 	Recorder     record.EventRecorder
 }
 
-var ComponentToConditionType = map[goharborv1.Component]goharborv1.HarborClusterConditionType{
-	goharborv1.ComponentHarbor:   goharborv1.ServiceReady,
-	goharborv1.ComponentCache:    goharborv1.CacheReady,
-	goharborv1.ComponentStorage:  goharborv1.StorageReady,
-	goharborv1.ComponentDatabase: goharborv1.DatabaseReady,
-}
+var (
+	ComponentToConditionType = map[goharborv1.Component]goharborv1.HarborClusterConditionType{
+		goharborv1.ComponentHarbor:   goharborv1.ServiceReady,
+		goharborv1.ComponentCache:    goharborv1.CacheReady,
+		goharborv1.ComponentStorage:  goharborv1.StorageReady,
+		goharborv1.ComponentDatabase: goharborv1.DatabaseReady,
+	}
+	ReconcileWaitResult = reconcile.Result{RequeueAfter: 30 * time.Second}
```

Comment:
Is it reasonable to use a fix 30 sec as the next reconcile time?

Reply:
yes, only in the case of returning an error, will wait for 30s.